apps/webview/src/hooks/useAuth.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { createContext, ReactNode, useContext, useMemo } from "react";
+import { useNavigate } from "react-router-dom";
+import { useLocalStorage} from "./useLocalStorage";
+
+
+interface AuthContextType {
```

Comment:
Move the types to a different directory named ```types``` to improve consistency and import the types from there.